packages/common/constants/servers.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * ⚠️ This file is in CommonJS format only.
+ * It is mainly used in Vite configuration (`vite.config.ts`).
+ * Due to use of ESbuild, Vite configuration only supports importing CommonJS modules.
+ *
+ * @see https://github.com/vitejs/vite/issues/7981
+ */
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
+const SERVERS = {
+  DEVELOPMENT: {
+    // @see: https://vitejs.dev/config/server-options.html
+    web: {
+      host: 'localhost',
+      port: 5172,
+      strictPort: true,
+    },
+    // @see: https://vitejs.dev/config/server-options.html
```

Comment:
this could either be at top level or at form-validations too.

Reply:
Fixed.